### PR TITLE
Update container plugin config.

### PR DESCRIPTION
**Pull request by dgoodwin**
Update for expected production directories to pull docker content from.

Deploy redhat-uep.crt in each of those default dirs.

### CODE REVIEW

File: subscription-manager.spec [hunk 1]
```diff
@@ -116,6 +116,13 @@ from the server. Populates /etc/docker/certs.d appropriately.
 %{_sysconfdir}/rhsm/pluginconf.d/container_content.ContainerContentPlugin.conf
 %{rhsm_plugins_dir}/container_content.py*
 %{_datadir}/rhsm/subscription_manager/plugin/container.py*
+# Copying Red Hat CA cert into each directory:
+%dir %{_sysconfdir}/docker/certs.d/cdn.redhat.com
+%dir %{_sysconfdir}/docker/certs.d/access.redhat.com
+%dir %{_sysconfdir}/docker/certs.d/registry.access.redhat.com
+%{_sysconfdir}/docker/certs.d/cdn.redhat.com
+%{_sysconfdir}/docker/certs.d/access.redhat.com
+%{_sysconfdir}/docker/certs.d/registry.access.redhat.com
```
Comment (alikins) [inline subscription-manager.spec]: python-rhsm sets the file and dir attrs specifically, ala:
%attr(755,root,root) %dir %{_sysconfdir}/rhsm/ca

%{python_sitearch}/rhsm/*
%{python_sitearch}/rhsm-*.egg-info
%attr(644,root,root) %{_sysconfdir}/rhsm/ca/*.pem

Suspect subman should as well.

Comment (alikins) [inline subscription-manager.spec]: For my test install, with selinux-policy-3.13.1-17.el7.noarch

I see different default context/labels for

```
/etc/rhsm/ca/redhat-uep.prm
[subscription-manager (dgoodwin/docker-config % u=)]$ ls -laZ /etc/rhsm/ca/redhat-uep.pem 
-rw-r--r--. root root system_u:object_r:etc_t:s0       /etc/rhsm/ca/redhat-uep.pem
[subscription-manager (dgoodwin/docker-config % u=)]$ matchpathcon /etc/rhsm/ca/redhat-uep.pem 
/etc/rhsm/ca/redhat-uep.pem	system_u:object_r:etc_t:s0
```

```
[subscription-manager (dgoodwin/docker-config % u=)]$ ls -laZ /etc/docker/certs.d/access.redhat.com/redhat-uep.crt 
-rw-r--r--. root root system_u:object_r:cert_t:s0      /etc/docker/certs.d/access.redhat.com/redhat-uep.crt
[subscription-manager (dgoodwin/docker-config % u=)]$ matchpathcon /etc/docker/certs.d/access.redhat.com/redhat-uep.crt
/etc/docker/certs.d/access.redhat.com/redhat-uep.crt	system_u:object_r:cert_t:s0
```

**Comment by dgoodwin** [comment]
Is this just because of the crt extension? Is it a problem?

**Comment by dgoodwin** [comment]
Updated to specify permissions on the cert.d directories, and redhat-uep.crt explicitly.

In the absence of more info I don't think the selinux labels are an issue, just the behaviour of the policy for files ending with crt. It's not something we explicitly set in either case correct?

**Comment by alikins** [comment]
ACK

**Pull request merged**